lib/OpenQA/Worker/Commands.pm

```diff
@@ -88,15 +88,15 @@ sub websocket_commands {
     elsif ($msg eq 'livelog_start') {
         # change update_status timer if $job running
         if (backend_running) {
+            print "starting livelog\n";
             $OpenQA::Worker::Jobs::do_livelog = 1;
-            change_timer('update_status', STATUS_UPDATES_FAST);
```

Comment:
can't you change the do_livelog to be a counter while you touch the code to fix the two visitors problem?